server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/GetMessageListMethodTest.java

```diff
@@ -240,6 +240,152 @@ public void getMessageListShouldFilterMessagesWhenMultipleInMailboxesFilterMatch
     }
 
```

Comment:
You should concider adding a test for when you have no filters (either inMailboxes and NotInMailboxes). I guess it is now supported.